cellrank/tl/kernels/_base_kernel.py

```diff
@@ -646,6 +648,13 @@ def _compute_transition_matrix(
             )
             matrix[problematic_indices, problematic_indices] = 1.0
 
+        if check_irreducibility:
```

Comment:
For consistency, I'd either name it as in base_estimator (`check_irred`), or vice-versa.

Reply:
I went for `check_irreducibility` and changed this in the `base_estimator`.